src/pages/help-centre/_help-articles.js

```diff
@@ -222,6 +434,40 @@ export const articles = [
                 label: 'withdrawal-processing-time',
                 content: <WithdrawalProcessingTime />,
             },
+            {
+                title: localize(
+                    'How do I withdraw funds from my Deriv account?',
+                ),
+                category: localize('Deposits and withdrawals'),
+                sub_category: localize('Withdrawals'),
+                label: 'How-do-I-withdraw-funds-from-my-Deriv-account',
+                content: <HowDoIWithdrawFundsFromMyDerivAccount />,
+            },
+        ],
+    },
+    {
+        category: localize('Privacy and security'),
+        articles: [
+            {
+                title: localize(
+                    'How we safeguard your account when you change or reset your Deriv password',
+                ),
+                category: localize('Privacy and security'),
+                sub_category: localize('Safeguard'),
+                label:
+                    'How-we-safeguard-your-account-when-you-change-or-reset-your-Deriv-password',
```

Comment:
can simplify this to safeguard-reset-password, and so do others

Reply:
Thank you Mustofa, if it is acceptable to make it shorter, so I will do.